Login fails on unmarshal of Nametag.SelfieSticker

```go
i := goinsta.New("<user>", "<pw>")
if err := i.Login(); err != nil {
	panic(err)
}
```

```
panic: json: cannot unmarshal string into Go struct field Nametag.selfie_sticker of type int
```

Any suggestions on how I could help to debug further...?

--- a/types.go
+++ b/types.go
@@ -68,7 +68,7 @@ type Nametag struct {
 	Mode          int64  `json:"mode"`
 	Gradient      int64  `json:"gradient"`
 	Emoji         string `json:"emoji"`
-	SelfieSticker int64  `json:"selfie_sticker"`
+	SelfieSticker string `json:"selfie_sticker"`
 }
 
 type friendResp struct {

